<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TextListItem</class>
 <widget class="QFrame" name="TextListItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>315</width>
    <height>82</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>315</width>
    <height>82</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>315</width>
    <height>82</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QFrame{
	background-color: rgb(236, 234, 232);
}
QFrame:hover{
	
	background-color: rgb(223, 221, 219);
}
QFrame:pressed{
	background-color: rgb(202, 200, 198);
}</string>
  </property>
  <widget class="QLabel" name="lb_pic">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>15</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(118, 118, 118);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lb_name">
   <property name="geometry">
    <rect>
     <x>89</x>
     <y>17</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;幼圆&quot;;
font-weight:500;
color: rgb(80, 80, 80);
background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string>你好</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_text">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>46</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 8.5pt &quot;幼圆&quot;;
font-weight:500;
color: rgb(120, 120, 120);
background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string>你好</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_date">
   <property name="geometry">
    <rect>
     <x>241</x>
     <y>16</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 8.5pt &quot;幼圆&quot;;
color: rgba(150, 150, 150,220);
font-weight:10pt;
background-color: rgba(255, 255, 255, 0);
</string>
   </property>
   <property name="text">
    <string>20/10/1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lb_bell">
   <property name="geometry">
    <rect>
     <x>281</x>
     <y>47</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="UsChat.qrc">:/Icon/Resources/Icon/bell_mute_small.png</pixmap>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="RedPoint">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>10</y>
     <width>14</width>
     <height>14</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 59, 48);
border-radius:7px;
min-width: 14px; 
min-height: 14px;
max-width:14px;
max-height: 14px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="UsChat.qrc"/>
 </resources>
 <connections/>
</ui>
